src/main/scala/cromwell/engine/workflow/WorkflowActor.scala

```diff
@@ -221,12 +221,19 @@ case class WorkflowActor(workflow: WorkflowDescriptor,
   /* Return a Markdown table of all entries in the database */
   private def symbolsMarkdownTable(): Option[String] = {
     val header = Seq("SCOPE","NAME","I/O","TYPE","VALUE")
```

Comment:
Those cool Markdown tables that are printed are entirely useless if they're too wide.